pyblish_qml/models.py

```diff
@@ -466,6 +466,9 @@ def update_with_result(self, result):
                 item.succeeded = True
                 item.amountPassed += 1
 
+            if not (item.hasWarning and not item.hasError):
```

Comment:
What does this do?

Reply:
Oh, it reset the warning, if this instance got error from any plugin.
Without this reset, warning will override the error state.

Reply:
Since this
```python
item.hasWarning |= any([ ...
```
lock the warning state, so need that

```python
if not (item.hasWarning and not item.hasError):
```
to take warning back for exposing the error state.

Reply:
Oh, sorry, that should be able simplify to
```python
if item.hasError:
    item.hasWarning = False
```